.drone.yml

```diff
@@ -99,6 +99,25 @@ pipeline:
     when:
       status:  success
 
+  vic-ui:
+    image: harbor.ci.drone.local/library/vic-integration-test:1.36
+    pull: true
+    environment:
+      BUILD_NUMBER: ${DRONE_BUILD_NUMBER}
+      BIN: bin
+      GS_PROJECT_ID: ${GS_PROJECT_ID}
+      GS_CLIENT_EMAIL: ${GS_CLIENT_EMAIL}
+      GS_PRIVATE_KEY: ${GS_PRIVATE_KEY}
+    commands:
+      - mkdir -p bin/ui
+      - wget -nv $(gsutil ls -l "gs://vic-ui-builds" | grep -v TOTAL | grep vic_ | sort -k2 -r | (trap " " PIPE; head -1) | xargs | cut -d " " -f 3 | sed "s/gs:\/\//https:\/\/storage.googleapis.com\//") -P bin/ui
+      - tar -xvzf bin/ui/vic_ui_*.tar.gz
+      - cp -rf bin/ci/ui/* bin/ui
```

Comment:
`ci` seems an unnecessary addition to the vic-ui build artifact. I see that https://github.com/vmware/vic-ui/pull/11 proposes to remove that folder, so just remember to update the `.drone.yml` file such that the script doesn't throw any non-zero status. This line though needs to be completely removed.